Fix some typos and confusing grammatical constructions.

stefanpf opened the pull request:
The German translation had a few typos and some grammatical constructions were a bit confusing. Tried to make things clearer.

stefanpf committed:
Fix some typos and confusing grammatical constructions.

Roshanjossey commented:
@lkreimann, could you please do a review on these changes? I'll share my thoughts as well.

lkreimann commented:
Will do :)

Roshanjossey commented on translations/README.de.md:
```diff
@@ -117,12 +124,11 @@ git push origin --delete <add-dein-Name>
-So wie du einen Pull Request gestellt hast, werden hoffentlich viele andere Entwickler einen Beitrag zum Projekt leisten. Sobald ein Pull Request angenommen wurde, sind die Änderungen im Master Branch des Projektes. In deinen Fork werden die Änderungen von anderen Freiwilligen aber nicht automatisch übernommen. Damit du immer die neuste Version hast, musst du regelmäßig synchronisieren. Wie dies  ...
+So wie du einen Pull Request gestellt hast, werden hoffentlich viele andere Entwickler einen Beitrag zum Projekt leisten. Sobald ein Pull Request angenommen wurde, sind die Änderungen im Master Branch des Projektes. In deinen Fork werden die Änderungen von anderen Freiwilligen aber nicht automatisch übernommen. Damit du immer die neueste Version hast, musst du regelmäßig synchronisieren. Wie dies ...
 
 ## Halte deinen Fork synchron
```
Could you please remove this section? Let's have the tutorial same as `Readme.md` and put everything else in `additional material`

Roshanjossey commented on translations/README.de.md:
```diff
@@ -166,7 +172,7 @@ Beachte, dass du hier zum Remote Namens origin pushst. Origin ist ein besonderer
-Hier sind einige Anfänger-Level Aufgaben (issues) in bekannten Repositories die du lösen kannst. Schau sie dir an und erfahre mehr darüber
+Hier sind einige Anfänger-Level-Aufgaben (issues) in bekannten Repositories, die du lösen kannst. Schau sie dir an und erfahre mehr darüber
 
 |[![exercism](https://avatars2.githubusercontent.com/u/5624255?v=3&s=100)](https://github.com/exercism/exercism.io/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+patch%22)|[![fun-retro](https://avatars3.githubusercontent.com/u/15913975?v=3&s=100)](https://github.com/funretro/distributed/issues?q=is%3Aopen+is%3Aissue+label%3Abeginner-friendly)|[<img width="100" src="https://cdn.worldvectorlog ...
```
Could you remove this section as well? We have this in our web app now.

Roshanjossey commented on translations/README.de.md:
```diff
@@ -57,40 +57,47 @@ Beispiel:
 
 Roshanjossey wird nun deine Änderungen in den Master Branch dieses Projekts mergen. Du erhältst eine E-Mail sobald dies geschehen ist. 
 
 ## Einen Branch aus deinem Repository löschen
```
Could you remove this section as well as per our discussion in #13435

Roshanjossey reviewed (changes requested):
I've pointed out a few changes that's required. Please address them.

stefanpf committed:
Changes due to change request

Deleted sections on deleting and synchronizing branches.

Deleted links to projects with first issues.

Re-arranged and updated the sections "Where to go from here?" and "Tutorials Using Other Tools" to mirror the English original.

Added section "Self-Promotion" at bottom to mirror English version.

stefanpf commented:
Hi @Roshanjossey, just wondering if this was still under consideration? I had made the changes you had requested in commit https://github.com/firstcontributions/first-contributions/pull/13436/commits/b42d902068d58789b54ebbe7fbfc1ca17608ade9 but not sure if you ever got a notification about this.

stefanpf commented:
Cool, thanks for the update!

lkreimann commented:
I'm gonna look over it in the evening. Sorry, wasn't sure if my review was still needed, since @Roshanjossey pointed out so much already!

Roshanjossey commented:
@lkreimann, no worries. Mein Duetsche ist immer noch nicht gut. I need expert review.

lkreimann commented on translations/README.de.md:
```diff
@@ -94,94 +94,29 @@ Erstelle einen Pull Request indem du auf die Schaltfläche `Create pull request`
-Wenn du weitere Fragen hast kannst du Mitglied in unserem Slackteam werden. [Join slack team](https://join.slack.com/t/firstcontributors/shared_invite/enQtMzE1MTYwNzI3ODQ0LTZiMDA2OGI2NTYyNjM1MTFiNTc4YTRhZTg4OWZjMzA0ZWZmY2UxYzVkMzI1ZmVmOWI4ODdkZWQwNTM2NDVmNjY).
+|<a href="github-desktop-tutorial.md"><img alt="GitHub Desktop" src="https://desktop.github.com/images/desktop-icon.svg" width="100"></a>|<a href="github-windows-vs2017-tutorial.md"><img alt="Visual Studio 2017" src="https://www.visualstudio.com/wp-content/uploads/2017/11/microsoft-visual-studio.svg" width="100"></a>|<a href="gitkraken-tutorial.md"><img alt="GitKraken" src="/assets/gk-icon.png" w ...
+|---|---|---|---|
+|[GitHub Desktop](github-desktop-tutorial.md)|[Visual Studio 2017](github-windows-vs2017-tutorial.md)|[GitKraken](gitkraken-tutorial.md)|[Visual Studio Code](github-windows-vs-code-tutorial.md)|
```
The paths here are incorrect - the files aren't located the translations directory. Look at the deleted code above - the relative paths there were correct. Just add a `../` at the beginning of the path. :)

lkreimann commented on translations/README.de.md:
````diff
@@ -94,94 +94,29 @@ Erstelle einen Pull Request indem du auf die Schaltfläche `Create pull request`
-```
-git fetch upstream
-```
+Wenn du weitere Fragen hast kannst du Mitglied in unserem Slack-Team werden. [Join Slack Team](https://join.slack.com/t/firstcontributors/shared_invite/enQtMzE1MTYwNzI3ODQ0LTZiMDA2OGI2NTYyNjM1MTFiNTc4YTRhZTg4OWZjMzA0ZWZmY2UxYzVkMzI1ZmVmOWI4ODdkZWQwNTM2NDVmNjY).
````
I guess after the word "hast" probably should be a comma. 🤔

lkreimann commented on translations/README.de.md:
```diff
@@ -8,37 +8,37 @@
 
 
-Wenn Git noch nicht installiert ist, [ installiere es ]( https://help.github.com/articles/set-up-git/ )
+Wenn du Git noch nicht installiert ist, [ installiere es ]( https://help.github.com/articles/set-up-git/ )
```
... installiert **hast**, installiere es?

lkreimann reviewed (changes requested):
Very good overall, just some small things that should be fixed before the merge :) Thanks for your work @stefanpf !

stefanpf committed:
Fixed two typos and broken links

stefanpf commented:
Hi @lkreimann, thanks for taking a look, I fixed the things you pointed out. Sorry for the long turn-around on this, I was on vacation :)

Roshanjossey committed:
Merge branch 'master' into fix-typos-in-german-translation

Roshanjossey commented:
Thank you for making these changes @stefanpf :vulcan_salute: 

Thank you for the review @lkreimann
